onnx2keras

bang6319 opened the pull request:
Hey Sarah,

Here's my onnx2keras feature branch

bang6319 committed:
my first commit

bang6319 committed:
my second commit

sarahaguasvivas commented on nn4mc/parser/onnx_parser/onnx_helpers.py:
```diff
@@ -0,0 +1,14 @@
+from onnx2keras import onnx_to_keras
+
+def onnx2keras(file):
+    # Load ONNX model
```
Please add more comments on what the function does using this template:

```
        /*------------------------------------------------- FUNCTION_NAME -----
         |  Function FUNCTION_NAME
         |
         |  Purpose:  EXPLAIN WHAT THIS FUNCTION DOES TO SUPPORT THE CORRECT
         |      OPERATION OF THE PROGRAM, AND HOW IT DOES IT.
         |
         |  Parameters:
         |      parameter_name (IN, OUT, or IN/OUT) -- EXPLANATION OF THE
         |              PURPOSE OF THIS PARAMETER TO THE FUNCTION.
         |                      (REPEAT THIS FOR ALL FORMAL PARAMETERS OF
         |                       THIS FUNCTION.
         |                       IN = USED TO PASS DATA INTO THIS FUNCTION,
         |                       OUT = USED TO PASS DATA OUT OF THIS FUNCTION
         |                       IN/OUT = USED FOR BOTH PURPOSES.)
         |
         |  Returns:  IF THIS FUNCTION SENDS BACK A VALUE VIA THE RETURN
         |      MECHANISM, DESCRIBE THE PURPOSE OF THAT VALUE HERE.
         *-------------------------------------------------------------------*/
```

sarahaguasvivas commented on nn4mc/parser/onnx_parser/_onnxparser.py:
```diff
@@ -0,0 +1,27 @@
+from nn4mc.parser._parser import Parser
```
TODO(sarahaguasvivas): Use `test_translator` to see if nn4mc doesn't crash with these models:

https://github.com/correlllab/nn4mc_py/blob/master/tests/test_translator/test_full.py

Create a new one called `test_full_onnx.py`

sarahaguasvivas reviewed (changes requested):
Very good work, Bao,

Now, let's see if it integrates well with the rest of the software

bang6319 committed:
+ Fixed model config bug - need to solve weights bug

bang6319 committed:
Update: ONNX parseWeights

Hi,

Here is the def parseWeights that I made for _onnxparser.py based on one from HDF5.

Bao

bang6319 committed:
my latest update

bang6319 committed:
Update model.hdf5
